web/docs/database/backends.md

```diff
@@ -1,3 +1,329 @@
 ---
-title: Database Backends
+title: Databases
 ---
+
+[Entities](/docs/database/entities.md), [Operations](/docs/database/operations.md) and [Automatic CRUD](/docs/database/crud.md) together make a high-level interface for working with your app's data. Still, all that data has to live somewhere, so let's see how Wasp deals with databases.
+
+## Supported Database Backends
+
+Wasp supports multiple database backends. We'll list and explain each one.
+
+### SQLite
+
+The default database Wasp uses is [`SQLite`](https://www.sqlite.org/index.html).
+
+SQLite is a great way for getting started with a new project because it doesn't require any configuration, but Wasp can only use it in development. Once you want to deploy your Wasp app to production, you'll need to switch to `PostgreSQL` and stick with it.
```

Comment:
```suggestion
SQLite is a great way for getting started with a new project because it doesn't require any configuration, but Wasp can only use it in development. Once you want to deploy your Wasp app to production, you'll need to switch to PostgreSQL and stick with it.
```